### PR TITLE
Nightly- Fixes some errors

### PR DESCRIPTION
**Pull request by eloycoto**
Hi, 

Fixed few errors that has been fired in the last Nightly PR: 

- Issues on retrieve pods (Due the high load, it's a Kubernetes resource, no Cilium related)
- Issues with policy templates. I have seen this two times this week. 
- Added logs in `kub.CiliumPolicyRevision` that it was fired one time and I'm not sure why. With this change we can see the log output instead `jq` output.

### CODE REVIEW

**Comment by eloycoto** [comment]
test-nightly

**Review by ianvernon — APPROVED** [review]

File: test/helpers/policygen/models.go [hunk 1]
```diff
@@ -477,9 +480,9 @@ func (t *TestSpec) CreateCiliumNetworkPolicy() (string, error) {
 	}
 
 	type Spec struct {
-		EndpointSelector endpointSelector
```
Comment (aanm) [inline test/helpers/policygen/models.go]: Why not use `pkg/policy/api/rule.go:Rule` directly instead of `Spec`?

**Review by aanm — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-nightly

File: test/helpers/policygen/models.go (hunk 1 above)
Comment (eloycoto) [reply test/helpers/policygen/models.go]: done.

**Comment by eloycoto** [comment]
test-nightly

**Review by aanm — APPROVED** [review]

**Comment by eloycoto** [comment]
test-nightly

**Comment by eloycoto** [comment]
test-nightly

**Comment by eloycoto** [comment]
test-nightly

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
nightly tests passed :) waiting for Ginkgo PR tests to pass too since the Vagrantfile was updated.

**Pull request merged**